src/components/pages/AccountPage/AccountPage.js

```diff
@@ -145,13 +140,7 @@ const AccountPage = props => {
                 onFinish={onEditSubmit}
               >
                 <h2 className="h1Styles">Edit User</h2>
-                <p className="divider">
-                  <Icon
-                    component={() => (
-                      <img src={OrangeLine} alt="divider icon" />
-                    )}
-                  />
-                </p>
+                <div className="divider" />

```

Comment:
Good call on changing the p tag for the divider, especially for accessibility reasons

Reply:
Seconding this - really good catch here!